report/web.py

```diff
@@ -148,7 +148,9 @@ def generate(self):
       self._write_benchmark_crash(benchmark, samples)
 
       for sample in samples:
-        sample_targets = self._results.get_targets(benchmark.id, sample.id)
+        sample_targets = (self._results.get_targets(benchmark.id, sample.id) or
```

Comment:
it's a bit confusing we have two very similar but different functions here. do we intend to get rid of the old one?